osf/management/commands/sync_datacite_doi_metadata.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import logging
+
+from django.core.management.base import BaseCommand
+from osf.models import Registration
+
+logger = logging.getLogger(__name__)
+
+
+def sync_datacite_doi_metadata(dry_run=True):
+    for registration in Registration.objects.all():
+        if not dry_run:
+            doi = registration.request_identifier('doi')
+            registration.set_identifier_value('doi', doi)
```

Comment:
Can this be a dry run if the DOI is minted?

Reply:
good point!

Reply:
Nothing at all is happening here during a dry run, so I'm confused.

Reply:
Yeah it's pretty pointless, but usually MattF like a dry run in case he wants to add additional logging statements if there is rate limiting problems, but there isn't a particular reason in this case it must have one.